packages/data/src/namespace-store/index.js

```diff
@@ -151,11 +178,15 @@ function mapActions( actions, store ) {
  *
  * @param {Object} resolvers   Resolvers to register.
  * @param {Object} selectors   The current selectors to be modified.
- * @param {Object} fulfillment Fulfillment implementation functions.
  * @param {Object} store       The redux store to which the resolvers should be mapped.
  * @return {Object}            An object containing updated selectors and resolvers.
  */
-function mapResolvers( resolvers, selectors, fulfillment, store ) {
+function mapResolvers( resolvers, selectors, store ) {
+	const mappedResolvers = mapValues( resolvers, ( resolver ) => {
+		const { fulfill: resolverFulfill = resolver } = resolver;
+		return { ...resolver, fulfill: resolverFulfill };
+	} );
```

Comment:
there are multiple ways to provide "fulfill" (this existed before, I just moved it because the implementation changed a little bit).

you can write 

`resolver = { fulfill: () => { }, isFullfilled: () => { } }`

or


```
resolver = function() {}
resolver.isFullfilled = () => {}
```

Reply:
this code normalizes all these approaches into one, that way the following code don't need "ifs"